docs/source/customization.rst

```diff
@@ -0,0 +1,68 @@
+
+=====================
+ Customizing the API
+=====================
+
+Content serialization
+=====================
+
+Dexterity fields
+----------------
+
+The API automatically converts all field values to JSON compatible
+data, whenever possible.
+However, you might use fields which store data which cannot automatically
```

Comment:
`store data which cannot automatically be converted` -> `store data that cannot be automatically converted,`